src/space.cpp

```diff
@@ -129,6 +129,11 @@ static argval_t do_space(chunk_t *first, chunk_t *second, int& min_sp, bool comp
       log_rule("IGNORED");
       return(AV_REMOVE);
    }
+   if ((first->type == CT_PP_IGNORE) && (second->type == CT_PP_IGNORE))
+   {
+	   log_rule("PP_IGNORE");
```

Comment:
needs some kind of comment about how CT_PP_IGNORE doesn't make sense as something we need to adjust spacing for. the point of it is to ignore blocks of preprocessor code, and definitely don't want the default fallthrough behavior.